go_modules/spec/dependabot/go_modules/update_checker/latest_version_finder_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require "dependabot/dependency"
+require "dependabot/dependency_file"
+require "dependabot/go_modules/native_helpers"
+require "dependabot/go_modules/update_checker/latest_version_finder"
+
+RSpec.describe Dependabot::GoModules::UpdateChecker::LatestVersionFinder do
+  let(:dependency_name) { "github.com/dependabot-fixtures/go-modules-lib" }
```

Comment:
For context: we are planning to move the `#latest_resolvable_version` tests from https://github.com/dependabot/dependabot-core/blob/c639755a862939c8d103898daadf8b63a2e85a10/go_modules/spec/dependabot/go_modules/update_checker_spec.rb#L55 to here so it makes sense to have this as the default even though it's not used.